site/content/3.12/operations/administration/configuration.md

```diff
@@ -330,10 +330,16 @@ instance, or adjusted at runtime via an API call. Examples:
 
 ## Fetch Current Configuration Options
 
-To list the configuration options of a running `arangod` instance, you can
-connect with an [ArangoShell](../../components/tools/arangodb-shell/_index.md) and invoke a
-[Transaction](../../develop/transactions/_index.md) by calling `db._executeTransaction()`
-and providing a JavaScript function to retrieve the server options:
+Use the [`GET /_admin/options` HTTP API](../../develop/http-api/administration.md#startup-options)
+to retrieve the effective configuration of the queried _arangod_ instance as
+set by startup options on the command-line and via a configuration file.
+
+Alternatively, you can connect to an `arangod` instance with an
+[ArangoShell](../../components/tools/arangodb-shell/_index.md) and invoke a
+[JavaScript Transaction](../../develop/transactions/javascript-transactions.md)
+by calling `db._executeTransaction()` with the following function to retrieve the
+server options, but this is limited to Coordinators and single servers with
+JavaScript/V8 enabled:
```

Comment:
I think we should stop recommending the JavaScript variant, as now a proper alternative is available.

Reply:
My thought was that people may look at the eventually stable 3.12 documentation and try to use the HTTP API but it's not available in 3.11 and older, and keeping the old variant allows me to reference it in the release notes - I generally avoid linking between versions. But I think we can make an exception here.